src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,114 @@
+.product-card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__image {
+    padding-inline: 3px;
+
+    & img {
+      width: 100%;
+      height: 134px;
+      background-size: cover;
```

Comment:
The `background-size: cover;` property is not applicable to `img` elements. It's used for the background images. If you want to control the size of an `img` element, use width and height properties.